### PR TITLE
wip: Add storybook

### PR DESCRIPTION
**Pull request by iamareebjamal**

### CODE REVIEW

File: stories/1-Button.stories.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Button',
+};
+
+export const Text = () => ({
+  template: hbs`<button {{action onClick}}>Hello Button</button>`,
+  context: {
```
Comment (niranjan94) [inline stories/1-Button.stories.js:10]: Codacy Issue found: [Missing space after key 'context'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172&bid=17608964)

File: stories/0-Welcome.stories.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export const ToStorybook = () => ({
+  template: hbs`
+    <div>
+      <h3> Welcome to Storybook! </h3>
+      <button {{action onClick}}> Checkout the button example </button>
+    </div>
+  `,
+  context: {
+    onClick: linkTo('Button'),
+  },
```
Comment (niranjan94) [inline stories/0-Welcome.stories.js:17]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+export const Emoji = () => ({
+  template: hbs`
+    <button {{action onClick}}>
+      <span role="img" aria-label="so cool">
+        😀 😎 👍 💯
+      </span>
+    </button>
+  `,
+  context: {
+    onClick: action('clicked'),
```
Comment (niranjan94) [inline stories/1-Button.stories.js:24]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Button',
+};
+
+export const Text = () => ({
+  template: hbs`<button {{action onClick}}>Hello Button</button>`,
```
Comment (niranjan94) [inline stories/1-Button.stories.js:9]: Codacy Issue found: [Missing space after key 'template'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/0-Welcome.stories.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Welcome',
```
Comment (niranjan94) [inline stories/0-Welcome.stories.js:5]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/0-Welcome.stories.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+export const ToStorybook = () => ({
+  template: hbs`
+    <div>
+      <h3> Welcome to Storybook! </h3>
+      <button {{action onClick}}> Checkout the button example </button>
+    </div>
+  `,
+  context: {
+    onClick: linkTo('Button'),
```
Comment (niranjan94) [inline stories/0-Welcome.stories.js:16]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/0-Welcome.stories.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  },
+});
+
+ToStorybook.story = {
+  name: 'to Storybook',
```
Comment (niranjan94) [inline stories/0-Welcome.stories.js:21]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Button',
```
Comment (niranjan94) [inline stories/1-Button.stories.js:5]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  template: hbs`
+    <button {{action onClick}}>
+      <span role="img" aria-label="so cool">
+        😀 😎 👍 💯
+      </span>
+    </button>
+  `,
+  context: {
+    onClick: action('clicked'),
+  },
```
Comment (niranjan94) [inline stories/1-Button.stories.js:25]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 9
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172&bid=17608964)

File: stories/0-Welcome.stories.js (hunk 7 above)
Comment (niranjan94) [inline stories/0-Welcome.stories.js:21]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js (hunk 1 above)
Comment (niranjan94) [inline stories/1-Button.stories.js:10]: Codacy Issue found: [Missing space after key 'context'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/0-Welcome.stories.js (hunk 2 above)
Comment (niranjan94) [inline stories/0-Welcome.stories.js:17]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/0-Welcome.stories.js (hunk 5 above)
Comment (niranjan94) [inline stories/0-Welcome.stories.js:5]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js (hunk 3 above)
Comment (niranjan94) [inline stories/1-Button.stories.js:24]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js (hunk 4 above)
Comment (niranjan94) [inline stories/1-Button.stories.js:9]: Codacy Issue found: [Missing space after key 'template'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js (hunk 9 above)
Comment (niranjan94) [inline stories/1-Button.stories.js:25]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/0-Welcome.stories.js (hunk 6 above)
Comment (niranjan94) [inline stories/0-Welcome.stories.js:16]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js (hunk 8 above)
Comment (niranjan94) [inline stories/1-Button.stories.js:5]: Codacy Issue found: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           

Complexity increasing per file
==============================
- stories/1-Button.stories.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172&bid=17608964)

File: stories/1-Button.stories.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Button',
+  argTypes: {
+    children: { control: 'text' },
```
Comment (niranjan94) [inline stories/1-Button.stories.js:8]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+  },
+};
+
+const Template = (args) => ({
```
Comment (niranjan94) [inline stories/1-Button.stories.js:12]: Codacy found an issue: [Unexpected parentheses around single function argument.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+Text.args = {
+  children: 'Button',
+  actions: {
+    onClick: action('onClick'),
+  }
+};
+
+export const Emoji = Template.bind({});
+Emoji.args = {
+  children: '😀 😎 👍 💯',
```
Comment (niranjan94) [inline stories/1-Button.stories.js:27]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+export default {
+  title: 'Button',
+  argTypes: {
+    children: { control: 'text' },
+  },
+};
+
+const Template = (args) => ({
+  template: hbs`<button {{action "onClick"}}>{{children}}</button>`,
+  context: args,
```
Comment (niranjan94) [inline stories/1-Button.stories.js:14]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 14]
```diff
@@ -0,0 +1,59 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Button',
```
Comment (niranjan94) [inline stories/1-Button.stories.js:6]: Codacy found an issue: [Missing space after key 'title'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+const Template = (args) => ({
+  template: hbs`<button {{action "onClick"}}>{{children}}</button>`,
+  context: args,
+});
+
+export const Text = Template.bind({});
+Text.args = {
+  children: 'Button',
```
Comment (niranjan94) [inline stories/1-Button.stories.js:19]: Codacy found an issue: [Missing space after key 'children'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+  actions: {
+    onClick: action('onClick'),
```
Comment (niranjan94) [inline stories/1-Button.stories.js:21]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+
+export default {
+  title: 'Button',
+  argTypes: {
+    children: { control: 'text' },
+  },
+};
+
+const Template = (args) => ({
+  template: hbs`<button {{action "onClick"}}>{{children}}</button>`,
```
Comment (niranjan94) [inline stories/1-Button.stories.js:13]: Codacy found an issue: [Missing space after key 'template'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 18]
```diff
@@ -0,0 +1,59 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Button',
+  argTypes: {
+    children: { control: 'text' },
+  },
```
Comment (niranjan94) [inline stories/1-Button.stories.js:9]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 19]
```diff
@@ -0,0 +1,59 @@
...
+
+const Template = (args) => ({
+  template: hbs`<button {{action "onClick"}}>{{children}}</button>`,
+  context: args,
+});
+
+export const Text = Template.bind({});
+Text.args = {
+  children: 'Button',
+  actions: {
```
Comment (niranjan94) [inline stories/1-Button.stories.js:20]: Codacy found an issue: [Missing space after key 'actions'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js (hunk 13 above)
Comment (niranjan94) [inline stories/1-Button.stories.js:14]: Codacy found an issue: [Missing space after key 'context'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 20]
```diff
@@ -0,0 +1,59 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  title: 'Button',
+  argTypes: {
```
Comment (niranjan94) [inline stories/1-Button.stories.js:7]: Codacy found an issue: [Missing space after key 'argTypes'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

File: stories/1-Button.stories.js [hunk 21]
```diff
@@ -0,0 +1,59 @@
...
+
+export const ButtonWithLinkToAnotherStory = () => ({
+  template: hbs`
+    <button {{action "onClick"}}>
+      Go to Welcome Story
+    </button>
+  `,
+  context: {
+    actions: {
+      onClick: linkTo('example-introduction--page'),
```
Comment (niranjan94) [inline stories/1-Button.stories.js:54]: Codacy found an issue: [Unexpected trailing comma.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5372172)

**Comment by maze-runnar** [comment]
we are not capable of finishing what he started 😞

**Pull request closed without merging**